src/game/time_attack/results.c

```diff
@@ -240,8 +240,8 @@ u32 CreateTimeAttackResults(u32 finishTime)
 
     resultsCutScene->transform.width = 0;
     resultsCutScene->transform.height = +Q(1);
-    resultsCutScene->transform.x = 120;
-    resultsCutScene->transform.y = 120;
+    resultsCutScene->transform.x = (DISPLAY_WIDTH / 2);
+    resultsCutScene->transform.y = (DISPLAY_HEIGHT - 40);
```

Comment:
Are you sure these should be changed like that?

What do these two values do?

Reply:
this is meant for the medal position, it can be changed to be under the timer if need be

Reply:
Oh, alright!

Yeah, offsetting it based on the timer's position might be the way to go here.